onboarding-extension-requests/README.md

```
@@ -0,0 +1,100 @@
+```
+
+#### Fields
+
+| Field         | Type      | Description                                                             |
+| ------------- | --------- | ------------------------------------------------------------------------|
+| id            | String    | Unique identifier for the document.                                     |
+| userId        | String    | The id of the user who created the request.                             |
+| state         | String    | The state of the request like APPROVED, REJECTED, PENDING.              | 
+| oldEndsOn     | number    | Old deadline of the onboarding task in millisecond                      |
+| newEndsOn     | number    | New deadline of the onboarding task in millisecond                      |
+| message       | String    | The message provided by super user while approving or rejecting request.| 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)